unaligned_bam_to_bqsr/downsample_workflow.cwl

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: Workflow
+label: "downsample unaligned BAM and align"
+requirements:
+    - class: ScatterFeatureRequirement
+    - class: SubworkflowFeatureRequirement
+    - class: MultipleInputFeatureRequirement
+
+inputs:
+    bams:
+        type: File[]
+    readgroups:
+        type: string[]
+    reference:
+        type: string
+    dbsnp:
+        type: File
+        secondaryFiles: [.tbi]
+    mills:
+        type: File
+        secondaryFiles: [.tbi]
+    known_indels:
+        type: File
+        secondaryFiles: [.tbi]
+    downsample_probability:
+        type: float
+outputs:
+    cram:
+        type: File
+        outputSource: align_workflow/cram
```

Comment:
The output of [workflow.cwl](https://github.com/jasonwalker80/cancer-genomics-workflow/blob/c94744a90f28e755721e15cdb48cce4949cf9ec9/unaligned_bam_to_bqsr/workflow.cwl#L28) is `final_cram`.